package.json

```diff
@@ -57,5 +57,10 @@
     "ts-node": "~7.0.0",
     "tslint": "~5.11.0",
     "typescript": "~3.4.5"
+  },
+  "browser": {
+    "fs": false,
+    "path": false,
+    "os": false
```

Comment:
Technically this isn't needed right now, but it will be if I get images working on the document using fs. This prevents an ng build error that says fs is not found.